src/AppBundle/Service/AddLicenseContact.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace AppBundle\Service;
+
+
+use AppBundle\Entity\License;
+use GuzzleHttp\Client;
+use Exception;
+use GuzzleHttp\Exception\ClientException;
+
+class AddLicenseContact
+{
+    private $apiDc;
+    private $apiKey;
+    private $listMap;
+
+    public function __construct($apiKey, $listMap)
+    {
+        $this->apiKey = $apiKey;
+
+        $keyParts = explode('-', $this->apiKey);
+        $this->apiDc = $keyParts[count($keyParts)-1];
+
+        $listMapTmp = explode(';', $listMap);
+
+        $listMaps = array(count($listMapTmp));
+
+        for ($i = 0; $i < count($listMapTmp); $i++) {
+            $listMaps[$i] = explode(':',$listMapTmp[$i]);
+
+            if (count($listMaps[$i]) != 2) {
+                throw new Exception('Badly formed list_map in parameters.yml.');
+            }
+        }
+
+        $this->listMap = $listMaps;
```

Comment:
Absolutely not necessary, I'm just not familiar with YAML or PHP.

What happens there is first getting the datacenter id from API key, then seperating the list of plugin key / list IDs. When there is a new license for a given plugin key, the technical contact will be added to the mailchimp list with that ID.